japanese/path/path.md

```diff
@@ -1,6 +1,7 @@
 あなたは小道を進んでいった。丁寧に整備のされた小道だ。数分歩いていると、ひとりの庭師が木の剪定をしているところにでくわした。紺色の羽織と足袋という和風の格好をした庭師だった。
 
 あなたは：
-庭師の作業を見る
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.